pkg/resources/database.go

```diff
@@ -19,11 +19,13 @@ var databaseSchema = map[string]*schema.Schema{
 	"name": {
 		Type:     schema.TypeString,
 		Required: true,
+		Description: "Specifies the identifier for the database; must be unique database name.",
```

Comment:
If you could change it to `Specifies the identifier for the database; must be unique for your account` that would be great, other than that I think It's good to merge. Thanks again for the contribution :)

Reply:
Hi @sfc-gh-jcieslak done it!